### PR TITLE
chore: maintainer & reviewer changes for CAPA

### PR DESCRIPTION
**Pull request by richardcase**
This updates the reviewers & approvers for the CAPA jobs.

/cc @Ankitasw @dlipovetsky @vincepri

### CODE REVIEW

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/OWNERS [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 
 reviewers:
-- dthorsen
-- pydctw
-- shivi28
 - AverageMarcus
+- luthermonson
+- cnmcavoy
+- nrb
+- faiq
```
Comment (vincepri) [inline config/jobs/kubernetes-sigs/cluster-api-provider-aws/OWNERS:8]: Remove until @faiq can ask for membership?

Comment (faiq) [reply config/jobs/kubernetes-sigs/cluster-api-provider-aws/OWNERS:8]: Thanks for the reminder I just asked for membership here: https://github.com/kubernetes/org/issues/4031

Comment (richardcase) [reply config/jobs/kubernetes-sigs/cluster-api-provider-aws/OWNERS:8]: When this merges https://github.com/kubernetes/org/issues/4733 lets get this change in

**Pull request merged**